docs/unified/novacustom/initial-deployment.md

```diff
@@ -0,0 +1,201 @@
+        ```
+
+
+    1. Install Dasharo BIOS, replacing `[path]` with the path to the firmware
+       image you want to flash, e.g. `novacustom_ns5x_adl_full_v1.4.0.rom`
+
+        ```bash
+        flashrom -p ch341a_spi -w [path]
+        ```
+
+    1. Detach the WSON-8 probe.
+    1. Connect the primary battery and reconnect the CMOS battery
+    1. Power on the laptop. The laptop may shut down once after training the memory.
+
+    ### EC firmware installation
+
+    Currently, the latest flashrom release lacks support for flashing ITE
+    embedded controllers such as the ones present in NovaCustom laptops. Because
+    of this, we need to build flashrom from source. Run the following commands
+    on the target laptop:
+
+    1. Install build dependencies:
+
+        ```bash
+        apt update
+        apt upgrade
+        apt install git build-essential debhelper pkg-config libpci-dev libusb-1.0-0-dev libftdi1-dev meson
+        ```
+
+    1. Obtain source code:
+
+        ```bash
+        git clone https://github.com/dasharo/flashrom.git
+        cd flashrom
+        ```
+
+    1. Build flashrom:
+
+        ```bash
+        make
+        sudo make install
+        ```
+
+    1. Make a backup of the EC firmware:
+
+        > Remember to store the backup on the separate device in case flashing
+        > fails.
+
+        ```bash
+        flashrom -p ite_ec -r ec_backup.rom
+        ```
+
+    1. Install the EC firmware:
+
+        > Warning: After running this command, the internal keyboard and power
+        > button will stop responding until the device is power cycled (all
+        > power must be removed, including the internal battery). Be prepared
+        > to disconnect the battery after updating the EC.
+
+        > Alternatively, you can boot up the laptop without the battery
+        > connected and power it from an AC adapter. Then, once the EC update
+        > is complete, disconnect the AC adapter to power off the laptop.
+
+        Run the following command, replacing `[path]` with the path to the EC
+        firmware you want to flash, e.g. `novacustom_ns5x_adl_ec_v1.4.0.rom`
+
+        ```bash
+        flashrom -p ite_ec -w [path]
+        ```
+
+    Successful installation of Dasharo EC finishes the initial deployment
+    process.
+
+=== "11th Gen (Tiger Lake)"
+
+    The following instructions describe how to flash Dasharo, but if you are
+    interested in version v1.3.0 or higher, which is only compatible with
+    Dasharo EC Firmware, follow this instruction:
+    [Dasharo EC Transition](../../../common-coreboot-docs/dasharo_tools_suite/#dasharo-ec-transition).
+
+    If this is your first experience with DTS, first read its
+    [documentation](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/).
+    We recommend using DTS with the
+    [Bootable over network](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#bootable-over-network)
```

Comment:
Yes, but this is `initial deployment` guide. To have this feature I need to have the Dasharo installed in the first place, right?

Reply:
True, although one still can boot iPXE manually and boot DTS that way :)

Changed to bootable usb stick

Reply:
@mkopec 

> True, although one still can boot iPXE manually and boot DTS that way :)

So we need to add those information on DTS site. (added as TODO here https://github.com/Dasharo/docs/pull/369)  BTW, please check links, that one https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#bootable-over-network leads to `404`. DTS docs were changed a bit so now it should be https://docs.dasharo.com/dasharo-tools-suite/documentation/#bootable-over-network

Reply:
> DTS docs were changed a bit

Yes, fixed already, see edited comment above